components/batch-system/src/batch.rs

```diff
@@ -479,15 +482,15 @@ impl<N: Fsm, C: Fsm, Handler: PollHandler<N, C>> Poller<N, C, Handler> {
             }
             self.handler.light_end(&mut batch.normals);
             for offset in &to_skip_end {
-                batch.schedule(&self.router, *offset, true);
+                batch.schedule(&self.router, *offset, true /* inplace */);
             }
             to_skip_end.clear();
             self.handler.end(&mut batch.normals);
 
             // Because release use `swap_remove` internally, so using pop here
-            // to remove the correct FSM.
+            // to remove the correct Fsm.
             while let Some(r) = reschedule_fsms.pop() {
-                batch.schedule(&self.router, r, false);
+                batch.schedule(&self.router, r, false /* inplace */);
```

Comment:
In fact, no matter using vscode or intellij, the arg name will be shown alongside with the value, the comment is a bit verbose.

Reply:
I don't think it's a good reason at all. GitHub review for one doesn't have plugins. And it doesn't make sense to intentionally "punish" those contributors (I for instance don't use them) without the necessary plugins.

Reply:
How about removing the bool parameter?